.bazelrc

```diff
@@ -28,6 +28,9 @@ common --experimental_cc_shared_library
 ## Unique configuration groups
 ###############################################################################
 
+# Enable use of the nightly toolchains by default.
```

Comment:
I don't see where we enable nightly toolchain by default though. Do we use `--config=nightly` by default anywhere?

Reply:
It's used by me every time I bump the Rust version 😅

Reply:
Can we reword the comment to clarify?

Reply:
Is this any less clarifying then the comments for clippy and rustfmt? While those are used in CI the comments don't refer to their use. Just what they do. Is this not following a similar pattern? Should they all be updated?

Reply:
IMO "by default" here is kinda misleading. It's not applied unless we run `bazel build --config=nightly`.

So unless we have

```
build (or common) --//rust/toolchain/channel=nightly
```

it's not really the default.

So just `enable use of the nightly toolchains` (without `by default`) is more accurate.

Reply:
Done!